src/Segmentation/LevelSets/SegmentWithGeodesicActiveContourLevelSet/Code.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<p>Point picker for itkwidgets would come in handy here for interactive experimentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/ITKSphinxExamples/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yes, on the agenda :-).